server/src/test/java/io/crate/integrationtests/AlterTableRenameColumnIntegrationTest.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.integrationtests;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static io.crate.testing.Asserts.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.elasticsearch.test.IntegTestCase;
+import org.junit.Test;
+
+import io.crate.exceptions.SQLParseException;
+import io.crate.testing.Asserts;
+import io.crate.testing.UseJdbc;
+
+public class AlterTableRenameColumnIntegrationTest extends IntegTestCase {
+
+    @Test
+    public void test_rename_primary_key_columns() {
```

Comment:
Do we really need to cover all scenarios with both the RenameTaskTest and integration test?

I think it would be safe to reduce this to one integration test - to ensure it's wired up correctly and to ensure the change is reflected in show create table and the system tables - and to rely on the `RenameTaskTest` for the rest?

Reply:
Thank you, reduced to a single test and move it to `AlterTableIntegrationTest`.